model_analyzer/result/run_config_measurement.py

```diff
@@ -520,4 +530,6 @@ def _deserialize_model_config_measurements(
             model_config_measurements.append(
                 ModelConfigMeasurement.from_dict(mcm_dict))
 
+            self._model_config_weights.append(1)
```

Comment:
won't this have real data?

Reply:
I've updated the code to restore these from the checkpoint and updated the L0's to use the new checkpoint that contains the default values.